platform/util/platform-util-unavailableurls/pom.xml

```diff
@@ -24,6 +26,11 @@
     <packaging>jar</packaging>
     <dependencies>
         <dependency>
+            <groupId>com.github.stefanbirkner</groupId>
```

Comment:
The dependency is for testing. It lets me set system properties for a test, then restores the original values. I'll add the test scope directive.

I have not run OWASP. I'll do that right after I go through the comments.